subtests/docker_cli/kill/kill_utils.py

```diff
@@ -129,7 +129,8 @@ def initialize(self):
         else:
             self._init_container_normal(name)
 
-        time.sleep(self.config['wait_start'])
+        cmd = self.sub_stuff['container_cmd']
+        cmd.wait_for_ready(timeout=self.config['wait_start'])
```

Comment:
you and me both, sadly I was sick of arguing with the guy who wrote that test so just told him to dupe it.  Those tests are much too complicated anyway, probably the best thing is to just scrap and re-write the important ones simpler.  Until someone has motivation for that...we duplicate code happily and enjoy our coffee/tea when working on the cleaner-parts :grinning: